### ISSUE TITLE
Right half marks are positioned wrong

### ISSUE THREAD

**Issue by dscorbett** [rsms/inter#327]
The right half marks U+FE21, U+FE23, U+FE28, and U+FE2A are positioned too far to the right.

**Screenshot**
This is <U+0078, U+FE20, U+0078, U+FE21>.
![x︠x︡](https://user-images.githubusercontent.com/1124347/96641800-4215a600-12f3-11eb-99de-f8a92074854e.png)

**Environment**
- OS: macOS 10.14
- Version of font: 3.15

**Comment by rsms** [comment]
Hm. I can't reproduce this. What software did you use to grab the screenshot?

This is what I see in Safari on macOS, which looks correct:

<img width="425" alt="Screen Shot 2021-03-28 at 14 04 14" src="https://user-images.githubusercontent.com/47046/112768009-a50c1700-8fce-11eb-8faa-2ad018566fed.png">

You can try it here: <https://rsms.me/inter/lab/?sample=%20%0Ax︠x︡%20%20X︠X︡&size=86>

**Comment by dscorbett** [comment]
I used hb-view (HarfBuzz). Safari uses Core Text. They differ because Core Text repositions the marks according to some internal heuristic, whereas HarfBuzz just puts them where the font says to. Either way, the rendering is wrong: the two halves should touch each other, as they do in DejaVu Sans:
![x︠x︡](https://user-images.githubusercontent.com/1124347/112770637-9422cc00-8ff5-11eb-9604-ae774c8ee603.png)

**Comment by rsms** [comment]
Thanks!

**Comment by VladWinner** [comment]
I confirm this on Windows 10 (Inter v3.19).

Chromium-based browsers:
![image](https://user-images.githubusercontent.com/13555921/124335359-b1da7100-dba2-11eb-8b1c-aa2af568cb62.png)

Firefox:
![image](https://user-images.githubusercontent.com/13555921/124335284-7dff4b80-dba2-11eb-82a3-05a498bc00ba.png)

FontLab 7 Preview window:
![image](https://user-images.githubusercontent.com/13555921/124335479-11388100-dba3-11eb-838f-8567eff4eb1e.png)

**Comment by Frondor** [comment]
I can also confirm in win10

Chrome @ 92.0.4515.131

![image](https://user-images.githubusercontent.com/5123709/129302687-f59d3d02-e6e8-4054-a9e0-4e99772a55dd.png)

**Comment by rsms** [comment]
What was going on:
The positioning of the actual glyphs U+FE20 and U+FE21 were correct ("touching each other" forming an arc.) They are however combining characters; the expectation is that when combined with another character, for example x, the curve would adjust to fit that character. This is known as "glyph composition/decomposition" described by the `ccmp` OpenType feature. While Inter has many ccmp rules to customized glyphs (for example A + grave accent is replaced by a specially designed "A with grave" glyph), it does not have rules for Combining Ligature Left Half et al. Supporting this would mean introducing potentially thousands of additional glyphs (one for every combination of what you might want to type next to a combining ligature line glyph.)

The following glyphs have been removed from Inter:

```
U+FE20  U+FE23  U+FE26  U+FE29  U+FE2C
U+FE21  U+FE24  U+FE27  U+FE2A  U+FE2D
U+FE22  U+FE25  U+FE28  U+FE2B
```

I'm not sure why I included these in the first place; they don't seem to be very useful.
